### ISSUE TITLE
Add two Green Bay buoys

### ISSUE THREAD

**Issue by kkoch** [glos/myglos#227]
Hi Everyone 
Not sure if GLOS would be interested or not but we have two new buoys in the lower Green Bay AOC this year as part of a long term project monitoring harmful algal blooms (and associated water quality conditions). I would be happy to send data to GLOS data portals.

See the following links:
http://www.lakestat.com/BuoyData/GBW/2018/Front2018.php
http://www.lakestat.com/BuoyData/GBE/2018/Front2018.php

Best,
Todd

Todd R. Miller, PhD
Associate Professor
Zilber School of Public Health
University of Wisconsin - Milwaukee
3135 N. Maryland Ave.
Room 5036
Milwaukee, WI 53211
443-255-0577
www.toddrexmiller.com

**Comment by kkoch** [comment]
PI Cookbook (enrollment guide) sent to Todd.  Also metadata form link.

**Comment by kkoch** [comment]
Green Bay East metadata > http://data.glos.us/metadata/srv/eng/main.home?uuid=e2d416bf-f370-4de9-8cc0-4e30338cb75c

Green Bay West metadata > http://data.glos.us/metadata/srv/eng/main.home?uuid=8fe994f1-27d1-4410-bb4e-05082938fc01

**Comment by kkoch** [comment]
@gcutrell : I've updated the links for 45184 and 45185 in GN and while I see data on GLBuoys, I do not see data in my json obs view (no parameters showing and no data when you click on the obs url) nor of course the portal itself.  

I did send Todd the instruction on him needing to send his data to NDBC for the texting on GLBuoys, but I would think that shouldn't prevent our display (esp. since I do see it on GLBuoys).  Right?
